cmd/root.go

```diff
@@ -45,7 +45,8 @@ func NewRootCmd() *cobra.Command {
 	}
 	houstonClient = houston.NewClient(httpClient)
 
-	astroClient := astro.NewAstroClient(httputil.NewHTTPClient())
+	// FIXME I think it is correct and recommended to share our HTTP client across all wrappers.
```

Comment:
Generally, it's ideal to have separate HTTP transport for each client because each client can have varying transport configurations/requirements. 

In this case, we have kept it separate because the configuration for `httpClient` is specific to the Houston client, and using the same for Astro would create confusion for the users as the configuration name would be quite misleading.